logos/services/README.md

```diff
@@ -3,3 +3,4 @@
 Create a `ID.png` PNG image with dimensions of 400x400 pixels, where `ID` is the ID of the service
 (you can see the ID in the service detail screen in the IO app).
```

Comment:
since we're modifying this already, I would add some suggestion on how to compress the file (perhaps with some online tool) - also add that the service ID [must be all lower case](https://github.com/teamdigitale/io-app/blob/master/ts/utils/services.ts#L24)